tests/x11regressions/firefox/sle12/firefox_flashplayer.pm

```diff
@@ -28,12 +28,17 @@ sub run() {
     assert_screen('firefox-flashplayer-verify_loaded', 45);
 
     send_key "pgdn";
-    assert_screen('firefox-flashplayer-verify', 25);
-
-    send_key "esc";
-    send_key "alt-d";
-    type_string "https://www.youtube.com/watch?v=Z4j5rJQMdOU\n";
-    assert_screen('firefox-flashplayer-video_loaded', 45);
+    # flashplayer dropped since sled12 sp2
+    while (check_screen([qw/firefox-flashplayer-dropped firefox-flashplayer-verify/])) {
```

Comment:
yes, but now you check_screen which might miss. What you really should do IMO is *assert* that one of the screens appears - i.e. fail if not. Just change check_screen to assert_screen should do that